crates/sui-core/src/unit_tests/batch_tests.rs

```diff
@@ -568,7 +568,7 @@ impl AuthorityAPI for TrustworthyAuthorityClient {
         let mut items = Vec::new();
         let mut last_batch = AuthorityBatch::initial();
         items.push({
-            let item = SignedBatch::new(last_batch.clone(), &*secret, name);
+            let item = SignedBatch::new(0, last_batch.clone(), &*secret, name);
```

Comment:
I'd create a static `ZERO_EPOCH = 0` variable just for readability and use this on all test functions that run with epoch = 0